html-web-page/package.json

```diff
@@ -1,13 +1,20 @@
 {
   "name": "romcal-html-web-page-example",
   "scripts": {
-    "start": "http-server ./"
+    "prestart": "cp node_modules/romcal/dist/iife/romcal.js public/romcal/romcal.js && cp node_modules/@romcal/calendar.france/iife/fr.js public/romcal/fr.js",
+    "start": "http-server -g -p 3000 ./public",
```

Comment:
I wish we could change the port using an env var (e.g. `ROMCAL_APP_PORT` or whatever env var name you prefer).

Reply:
as before, will do.